hw1103/src/main/java/com/example/hw1103/backend/User.java

```diff
@@ -0,0 +1,32 @@
+package com.example.hw1103.backend;
+
+public class User {
```

Comment:
How about to rewrite it on Kotlin data class?)

Reply:
done